src/operators/ops_generic.cpp

```diff
@@ -12,6 +12,7 @@
 #include "../runtime/d_array.h"
 #include "../runtime/diagnostics/d_stacktrace.h"
 #include "../runtime/d_code.h"
+#include "d_text.h"
```

Comment:
why?

Reply:
Used by `hint`

Reply:
but youre not using it with hint, you just added the include

Reply:
`runtime.register_sqfop(unary("hint", t_text(), "Outputs a hint message.", hint_text));`

Reply:
okay, but given that you introduced the new data class, you need to alter the existing too or it will cause issues in the long run (as the d_text class differs from the d_string class)
Please adjust the method accordingly.

Reply:
Good catch 👍